amplitude.js

```diff
@@ -320,7 +326,7 @@ Amplitude.prototype.setVersionName = function(versionName) {
   }
 };
 
-Amplitude.prototype.logEvent = function(eventType, eventProperties) {
+Amplitude.prototype.logEvent = function(eventType, eventProperties, apiProperties) {
```

Comment:
I'm wary about exposing a public call where the end user can mess with the apiProperties. Maybe make this internal and have an external facing logEvent that doesn't expose apiProperties?